# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/Cargo.toml
# Commit: b021b326fa47
# Full Hash: b021b326fa47eb57e14f858078ab5daaa6cd2274
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-06-20 03:53:45
# Regressor Bug: 1839070
# File Overlap Count: 6
# Description:
#   Bug 1839070 - Emit a start point if close is called immediately after move_to. r=jrmuizel
#   
#   wpf-gpu-raster would blindly modify the last point type in a path when call was closed, even
#   if move_to preceded it directly and outputted no actual point type. In this case, we need to
#   ensure that we emit a start point if close is called immediately after move_to.
# ==============================================================================

diff -r 4c4b0b9db374 -r b021b326fa47 toolkit/library/rust/shared/Cargo.toml
--- a/toolkit/library/rust/shared/Cargo.toml	Mon Jun 19 19:34:31 2023 +0000
+++ b/toolkit/library/rust/shared/Cargo.toml	Mon Jun 19 19:54:24 2023 +0000
@@ -94,7 +94,7 @@
 processtools = { path = "../../../components/processtools" }
 qcms = { path = "../../../../gfx/qcms", features = ["c_bindings", "neon"], default-features = false }
 
-wpf-gpu-raster = { git = "https://github.com/FirefoxGraphics/wpf-gpu-raster", rev = "a6514854d4518b02f2805719ff6cd74dae7dfde6" }
+wpf-gpu-raster = { git = "https://github.com/FirefoxGraphics/wpf-gpu-raster", rev = "112a2140c433af06938c663fc6777e2bf17bb6ce" }
 aa-stroke = { git = "https://github.com/FirefoxGraphics/aa-stroke", rev = "07d3c25322518f294300e96246e09b95e118555d" }
 
 # Force url to stay at 2.1.0. See bug 1734538.
